db/migration/20210530121921_create_shares_table.go

```diff
@@ -0,0 +1,34 @@
+package migrations
+
+import (
+	"database/sql"
+
+	"github.com/pressly/goose"
+)
+
+func init() {
+	goose.AddMigration(upCreateSharesTable, downCreateSharesTable)
+}
+
+func upCreateSharesTable(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+create table share
+(
+	id             varchar(255) not null primary key,
+	jwt            varchar(255) not null unique,
+	description    varchar(255),
+	expires        datetime,
+	created        datetime,
+	last_visited   datetime,
+	resource_id    varchar(255) not null,
```

Comment:
Also think a bit more in the future, this model (resource_id + resource_type) will make it harder to share a few selected songs. I suggest you rename this field to `resource_ids` and make it `varchar` (without size limit), so we can add a list of songs here

Reply:
Don't know why I thought that time that sharing multiple songs would be done via auto-generated playlists. 
Done now